src/components/filter/expert/ExpertFilterEditionDialog.tsx

```diff
@@ -108,7 +92,7 @@ export function ExpertFilterEditionDialog({
                 filterForm[EXPERT_FILTER_QUERY],
                 filterForm[FieldConstants.EQUIPMENT_TYPE],
                 filterForm[FieldConstants.NAME],
-                '', // The description can not be edited from this dialog
+                filterForm[FieldConstants.DESCRIPTION],
```

Comment:
```suggestion
                filterForm[FieldConstants.DESCRIPTION] ?? '',
```
to keep previous behaviour